File: SEBASTIAN_CHAN_00550218/C++/Gato.cpp
#include "Gato.hpp"
#include <string>
#include <iostream>
using namespace std;

// Constructor
Gato::Gato(string nombre, double peso, int b) : Mamifero(nombre, peso), nBigotes(b) {}

// Método para imprimir detalles del gato
void Gato::imprimirG(){
    cout << "Nombre del gato: " << getNombre() << endl;
    cout << "Peso: " << getPeso() << endl;
    cout << "Número de bigotes: " << nBigotes <<endl;
}

File: SEBASTIAN_CHAN_00550218/Pyth/Gato.py
from Mamifero import Mamifero
import tkinter as tk
from tkinter import messagebox

root=tk.Tk()
root.withdraw() 


class Gato(Mamifero):
    def __init__(self, nombre, peso, nBigotes):
        super().__init__(nombre, peso)
        self.nBigotes = nBigotes
    
    def imprimirG(self):
        messagebox.showinfo("Nombre del gato:", self.getNombre())
        messagebox.showinfo("Peso:", self.getPeso())
        messagebox.showinfo("Número de bigotes:", self.nBigotes)